x/ecocredit/spec/03_messages.md

```diff
@@ -14,6 +14,20 @@
 - `credit_type` (the name of the credit type) must not be empty and on the list of approved credit types (the `credit_types` parameter)
 - `metadata` must be less than or equal to 256 bytes
 
+## MsgCreateProject
+
+`MsgCreateProject` creates a new project with a credit class ID, issuer, project location, and optional metadata. The party signing this transaction is the credit admin. 
```

Comment:
isn't the party signing the tx one of the credit class issuers? see https://github.com/regen-network/regen-ledger/blob/baa7859987e2c714f4ca4106d7b764f881c38ab8/x/ecocredit/server/core/create_project.go#L23

Reply:
This is correct. I think this was a sloppy copy pasta.